iothub/device/src/Transport/AmqpIoT/AmqpIoTSession.cs

```diff
@@ -203,12 +203,14 @@ TimeSpan timeout
             string deviceTemplate,
             string moduleTemplate,
             string linkSuffix,
-            string CorrelationId,
+            string correlationId,
             TimeSpan timeout
         )
         {
             if (Logging.IsEnabled) Logging.Enter(typeof(AmqpIoTSession), deviceIdentity, $"{nameof(OpenSendingAmqpLinkAsync)}");
 
+            string serviceApiVersion = ClientApiVersionHelper.ApiVersionString;
+
```

Comment:
How would you set this to preview api version? Does ClientApiVersionHelper give you the right service api version?

Reply:
nm.. seeing it later

Reply:
it is being set to the preview API version below, based on the check if we have model Id set into client options.